### PR TITLE
refactor to have services with categories

### LINKED ISSUE #99: Re-Order alert form

### PR DESCRIPTION
**Pull request by nobane**
closes #99

### CODE REVIEW

File: _15thnight/api/service.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+from _15thnight.forms import ServiceForm
+from _15thnight.models import Service, Category
+from _15thnight.util import required_access, jsonify, api_error
+
+service_api = Blueprint('service_api', __name__)
+
+
+@service_api.route('/service', methods=['GET'])
+@login_required
+def get_categories():
```
Comment (justinnoah) [inline _15thnight/api/service.py]: Perhaps call this get_services? We have a get_categories in the categories api.

File: _15thnight/migrations/versions/368b199625d8_service_category_sort.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+# revision identifiers, used by Alembic.
+revision = '368b199625d8'
+down_revision = '9b6d9727e3ac'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
```
Comment (justinnoah) [inline _15thnight/migrations/versions/368b199625d8_service_category_sort.py]: unnecessary

File: _15thnight/migrations/versions/9b6d9727e3ac_services.py [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+# revision identifiers, used by Alembic.
+revision = '9b6d9727e3ac'
+down_revision = '4aa97e34f066'
+branch_labels = None
+depends_on = None
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mysql
```
Comment (justinnoah) [inline _15thnight/migrations/versions/9b6d9727e3ac_services.py]: unnecessary

File: client/src/actions/index.js [hunk 4]
```diff
@@ -499,15 +467,115 @@ export function resetPassword(data) {
...
+    let promise = request.get('/api/v1/service');
+    return dispatch => {
+        promise.then(
+            res => {
                 dispatch({
-                    type: APP_ERROR,
-                    message: 'An unknown error occured while resetting password.'
-                })
+                    type: GET_SERVICES,
+                    categories: res.data
```
Comment (justinnoah) [inline client/src/actions/index.js]: should this be called services now?

**Comment by nobane** [comment]
@justinnoah fixed all the feedback.

**Pull request merged**